deployment/v3/external/kafka/README.md

```diff
@@ -8,137 +8,93 @@
-        .........................................
-        Backup completed with status: Completed. You may check for more information using the commands `velero backup describe kafka-15-09-2022-13-21` and `velero backup logs kafka-15-09-2022-13-21`.
-  ```
-
-
-#### Restore Kafka
-* Ensure `values.yaml` & `ui-values.yaml` is up-to-date.
-* Ensure to update the required storage in `values.yaml`.
-* Run `restore.sh` to restore kafka from backup
-  ```
-    $ ./restore.sh  <K8S_CLUSTER_CONFIG_FILE>
-
-     ============== Check Cluster Config File ==========================================================================================================
-
-      Kubernetes Cluster file found 
-
-     ============== Check packages installed ===========================================================================================================
-
-      kubectl, minio client (mc), & velero packages are already installed !!! 
-
-     ============== S3 Setup ===========================================================================================================================
-      Provide S3 server : <S3_SERVER_NAME>
-      Provide S3 access key : <S3_ACCESS_KEY>
-      Provide S3 secret key : <S3_SECRET_KEY>
-      Provide S3 region ( Default region = minio ) : <S3_REGION>
-     ....
-     ....
-     ============== Create Restore =====================================================================================================================
-      Provide Namespace to restore ( Default Namespace = kafka ) : <NAMSPACE_TO_RESTORE>
-     ....
-     ....
-     ============== Restore Kafka from Backup ==========================================================================================================
-     [ List Backups ] 	
-      NAME                     STATUS      ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
-      kafka-backup             Completed   0        0          2022-08-25 20:00:08 +0530 IST   2d        default            app.kubernetes.io/instance=kafka
-      kafka-backup-1           Completed   0        0          2022-08-29 15:03:31 +0530 IST   6d        default            app.kubernetes.io/instance=kafka
-     [ Check backup existence ]
-      Provide Backup Name : <KAFKA_BACKUP_NAME>
-  ```
-
-## Troubleshoot
-* If you encounter the following error while executing `backup.sh` or `restore.sh`.
-  ```
-  mc: <ERROR> Unable to list folder. The request signature we calculated does not match the signature you provided. Check your key and signing method
-  ```
-  Then, ensure that you provide the appropriate API version, either `--api S3v2` or `--api S3v4`, when executing the `mc set alias` command.
-* To list backup/restore.
-  ```
-  velero --kubeconfig <K8S_CLUSTER_CONFIG_FILE> get backup
+#### Backup
+* Set WebSub and Kafka replicas to zero via the below command.
```

Comment:
Set WebSub and Kafka replicas to zero for V2 based existing env use below command.